chart/skywalking/values.yaml

```diff
@@ -97,6 +97,7 @@ ui:
     # clusterIP: None
     externalPort: 80
     internalPort: 8080
+    nodePort: 30080
```

Comment:
I think we recommend the users to configure a ingress gateway instead of expose a node port, no? @hanahmily

Reply:
nodePort seems a default value to the port number once the service type is `NodePort`, but`service.type` determines the service type, defaulting to `ClusterIP`. 

and I prefer to use ingress to expose UI service, let's set `ingress.enabled` to `true`

Reply:
I added a sample configuration for `nodePort` because it's used in the service `skywalking-kubernetes/chart/skywalking/templates/ui-svc.yaml`. Some users may choose `F5+NodePort` to deploy services instead of Ingress